src/snapred/ui/workflow/NormalizationCalibrationWorkflow.py

```diff
@@ -93,11 +93,19 @@ def _triggerNormalizationCalibration(self, workflowPresenter):
         self.backgroundRunNumber = view.getFieldText("backgroundRunNumber")
         self.sampleIndex = (view.sampleDropDown.currentIndex()) - 1
         self.initGroupingIndex = (view.groupingFileDropDown.currentIndex()) - 1
-        self.initSmoothingParameter = float(view.getFieldText("smoothingParameter"))
         self.samplePath = view.sampleDropDown.currentText()
         self.groupingPath = view.groupingFileDropDown.currentText()
         self.initDMin = float(self._specifyNormalizationView.fielddMin.field.text())
 
+        payload = NormalizationCalibrationRequest(
+            runNumber=self.runNumber,
+            backgroundRunNumber=self.backgroundRunNumber,
+            calibrantSamplePath=str(self.samplePaths[self.sampleIndex]),
+            focusGroup=self.focusGroups[str(self.groupingFiles[self.initGroupingIndex])],
+            crystalDMin=self.initDMin,
+        )
+        self.initSmoothingParameter = payload.smoothingParameter
+
```

Comment:
Is there any particular reason why this was moved?

Reply:
to pull the `self.initSmoothingParameter` from it as it gets initialized with the default value.